src/test/ui/test-attrs/test-fn-signature-verification-for-explicit-return-type.rs

```diff
@@ -1,4 +1,5 @@
 // run-pass
+// ignore-fuchsia Test must be run out-of-process
```

Comment:
Should this be `needs-unwind`? Or I don't understand what the comment means, anyway. I'm not sure if bench works with panic=abort.